### PR TITLE
Add SVGMetadata API

### PR DESCRIPTION
**Pull request by haydar-metin**
This PR is part of my diploma research conducted at the [Model Engineering lab](https://me.big.tuwien.ac.at/) of TU Wien in the [Business Informatics Group](https://www.big.tuwien.ac.at/).

---

It adds the necessary SVGMetadataAPI for `glsp-playwright`.

https://github.com/eclipse-glsp/glsp-playwright/pull/1

Contact: @martin-fleck-at, @planger, @ndoschek

### CODE REVIEW

**Comment by martin-fleck-at** [comment]
@haydar-metin I'll try to have a look a bit later that week. Could you please resolve the merge conflicts? Thank you!

File: examples/workflow-glsp/src/di.config.ts [hunk 1]
```diff
@@ -89,5 +90,8 @@ export function initializeWorkflowDiagramContainer(
         baseDiv: widgetId,
         hiddenDiv: widgetId + '_hidden'
     });
+
+    container.load(svgMetadataModule);
```
Comment (martin-fleck-at) [inline examples/workflow-glsp/src/di.config.ts:94]: I think this module is generic and useful enough to be added to the DEFAULT_MODULES.

File: packages/client/src/features/svg-metadata/metadata-placer.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/********************************************************************************
+ * Copyright (c) 2020-2022 EclipseSource and others.
```
Comment (martin-fleck-at) [inline packages/client/src/features/svg-metadata/metadata-placer.ts:2]: See other copyright header comment.

**Review by martin-fleck-at — CHANGES_REQUESTED** [review]
Hi @haydar-metin, the change looks good to me technically. I'd even suggest to put the module in the general default modules as it just adds some metadata. However, I think we need to adapt the copyright header - at least the year - and maybe the copyright owner as well.

File: packages/client/src/features/svg-metadata/di.config.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+/********************************************************************************
+ * Copyright (c) 2020-2022 EclipseSource and others.
```
[the remark this answers is not in the document]
Comment (haydar-metin) [reply packages/client/src/features/svg-metadata/di.config.ts:2]: Not really, however, in my proposal it says that the whole thesis will be contributed to EclipseSource, therefore I do not think that any problems will arise.

File: packages/client/src/features/svg-metadata/di.config.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { ContainerModule } from 'inversify';
+import '../../../css/decoration.css';
```
Comment (martin-fleck-at) [inline packages/client/src/features/svg-metadata/di.config.ts:18]:
```suggestion
```

File: packages/client/src/features/svg-metadata/di.config.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+import { TYPES } from '../../base/types';
+import { MetadataPlacer } from './metadata-placer';
+
+const svgMetadataModule = new ContainerModule((bind, _unbind, isBound) => {
```
Comment (martin-fleck-at) [inline packages/client/src/features/svg-metadata/di.config.ts:22]:
```suggestion
const svgMetadataModule = new ContainerModule(bind => {
```

**Review by martin-fleck-at — CHANGES_REQUESTED** [review]
Thank you for the update! I noticed to more things now that I had another look. And just to confirm: Having the copyright contribution as 'EclipseSource and others' is fine?

**Comment by martin-fleck-at** [comment]
@haydar-metin If you confirm the copyright, I think we can merge this change.

**Review by martin-fleck-at — APPROVED** [review]
Thank you for the update! LGTM!

**Pull request merged**